MyCapytain/resources/prototypes/capitains/collection.py

```diff
@@ -0,0 +1,477 @@
+            strings.append(make_xml_node(self.graph, RDF_NAMESPACES.CAPITAINS.term("structured-metadata")))
+            strings += structured_strings
+            strings.append(
+                    make_xml_node(self.graph, RDF_NAMESPACES.CAPITAINS.term("structured-metadata"), close=True)
+                )
+            strings.append(make_xml_node(self.graph, self.TYPE_URI, close=True))
+
+            return lines.join(strings)
+        elif output == Mimetypes.PYTHON.ETREE:
+            return xmlparser(self.export(output=Mimetypes.XML.GUIDELINES3, recursion_depth=recursion_depth))
+
+
+class CapitainsReadableMetadata(ResourceCollection, PrototypeCapitainsCollection):
+    """ Represents a CAPITAINS CapitainsTextMetadata
+
+    :param urn: Identifier of the CapitainsTextMetadata
+    :type urn: str
+    :param parent: Item parents of the current collection
+    :type parent: [PrototypeCapitainsCollection]
+
+    :ivar urn: URN Identifier
+    :type urn: URN
+    """
+
+    DC_TITLE_KEY = RDF_NAMESPACES.CAPITAINS.term("title")
+    MODEL_URI = URIRef(RDF_NAMESPACES.DTS.resource)
+    EXPORT_TO = [Mimetypes.XML.GUIDELINES3]
+    CAPITAINS_PROPERTIES = [RDF_NAMESPACES.CAPITAINS.identifier, RDF_NAMESPACES.CAPITAINS.parent]
+    CAPITAINS_LINKS = [RDF_NAMESPACES.CAPITAINS.about]
+
+    def __init__(self, urn="", parent=None, lang=None):
+        super(CapitainsReadableMetadata, self).__init__(identifier=str(urn))
+        self.resource = None
+        self.citation = None
+        self.__urn__ = str(urn)
+        self.docname = None
+        self.validate = None
+        self.lang = lang
+
+        if parent is not None:
+            self.parent = parent
+
+    @property
+    def readable(self):
+        return True
+
+    @property
+    def members(self):
+        """ Children of the collection's item
+
+        .. warning:: CapitainsCtsText has no children
+
+        :rtype: list
+        """
+        return list()
+
+    @property
+    def descendants(self):
+        """ Descendants of the collection's item
+
+        .. warning:: CapitainsCtsText has no Descendants
+
+        :rtype: list
+        """
+        return list()
+
+    def translations(self, key=None):
+        """ Get translations in given language
+
+        :param key: Language ISO Code to filter on
+        :return:
+        """
+        translations = []
+        for parent in self.parent:
+            translations.extend(parent.get_translation_in(key))
+        return translations
+
+    @property
+    def readable_siblings(self):
+        """ Get all readable siblings of the text
+
+        :return: List of readable siblings
+        :rtype: [CapitainsTextMetadata]
+        """
+        sibs = dict()
+        for parent in self.parent:
+            sibs.update({x.id: x for x in parent.readableDescendants})
+        sibs.pop(self.id, None)
+        return list(sibs.values())
+
+    def get_root_collection(self, lang=None):
```

Comment:
If we have parents plural (which mean there might be different ancestor roots), does not make any sense. Plus, this feels like making things easier for implementer but adding SO MUCH load for maintainers.